extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -387,9 +374,25 @@ export class Bind {
       }
       const element = dev().assertElement(node);
       const tagName = element.tagName;
-      if (DYNAMIC_TAGS[tagName]) {
-        this.observeElementForMutations_(element);
+
+      let dynamicElements = [];
+      if (typeof element.getDynamicElements === 'function') {
+        dynamicElements = element.getDynamicElements();
+      } else if (element.tagName === 'FORM') {
+        // FORM is not an amp element, so it doesn't have the getter.
```

Comment:
Probably need to add it to the sources list for Closure type checker in `build-system/tasks/compile.js`.